src/base-commands/twilio-client-command.js

```diff
@@ -26,6 +26,7 @@ class TwilioClientCommand extends BaseCommand {
 
     this.currentProfile = this.userConfig.getProfileById(this.flags.profile);
     let keytarFlag = false;
+    const helperLibrary = (this.config.userAgent || ' ').split(' ')[0];
```

Comment:
Why the term `helperLibrary` ?

Reply:
To make it a bit generalised. The const would contain the value of library/version (twilio-cli/2.27.1), I named it "extensions" initially but that sounded vague.